init.lua

```diff
@@ -8,8 +8,15 @@ local pvp_areas_modname = minetest.get_current_modname()
 
 local hasareasmod = minetest.get_modpath("areas")
 
-local safemode = minetest.setting_getbool("pvp_areas.safemode")
+local safemode = minetest.setting_getbool("pvp_areas.safemode") or false
```

Comment:
The `or false` needs removing, this was a bug that was fixed as per your initial report.

Reply:
if there is no lien in the minetest.conf with   pvp_areas.safemode
then safemode will be false? 
the minetst.setting_getbool return the bool or false if there is no flag in minetest.conf?